lib/src/domain/authorization/authentication_repository.dart

```diff
@@ -0,0 +1,13 @@
+import 'package:allthenews/src/domain/model/user.dart';
+
+abstract class AuthenticationRepository {
```

Comment:
pakiet z `lib/src/domain/authorization` na `lib/src/domain/authentication`

Reply:
do tej pory w authorization były rzeczy związane z kluczem api, jest to inny rodzaj autoryzacji niż logowanie od apki i dobrze by było to rozdzielić. Może obecne `authorization` zmienię na `apiauthentication` i zrobię osobny pakiet `authentication` na rzeczy związane z logowaniem/rejestracją?

Reply:
Te rzeczy co były wcześniej w authorization niech zostaną w authorization, niech powstanie osobny pakiet authentication i do niego wrzuć te rzeczy z logowaniem, bo ten api key to autoryzacja, a logowanie to autentykacja

Reply:
git, też o tym myślałem, ale zbyt podobne wydawały mi się te nazwy